### PR TITLE
Add UT test to nfs_test.go

### PR DESCRIPTION
**Pull request by jianglingxia**
**What type of PR is this?**
 /kind failing-test

**What this PR does / why we need it**:
add UT test to nfs_test.go
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jianglingxia** [comment]
PTAL @jingxu97
thanks

**Comment by jianglingxia** [comment]
/cc @msau42
thanks

File: pkg/volume/nfs/nfs_test.go [hunk 1]
```diff
@@ -245,3 +245,21 @@ func TestPersistentClaimReadOnlyFlag(t *testing.T) {
...
+		t.Fatalf("Can't make a temp dir: %v", err)
+	}
+	defer os.RemoveAll(tmpDir)
+	plugMgr := volume.VolumePluginMgr{}
+	plugMgr.InitPlugins(ProbeVolumePlugins(volume.VolumeConfig{}), nil /* prober */, volumetest.NewFakeVolumeHost(tmpDir, nil, nil))
+	plug, err := plugMgr.FindPluginByName("kubernetes.io/nfs")
+	if err != nil {
+		t.Errorf("can't find nfs plugin by name")
+	}
+	nfsSpec, err := plug.(*nfsPlugin).ConstructVolumeSpec("nfsVolume", "/nfsVolume/")
```
Comment (msau42) [inline pkg/volume/nfs/nfs_test.go]: Actually I think nfs `ConstructVolumeSpec` is completely wrong.  It's setting path to the volume name which doesn't sound right.  Maybe the correct thing to do here is to return an "unsupported" error.  cc @jingxu97

**Pull request closed without merging**